Volume/Supporting/UserData.swift

```diff
@@ -59,6 +65,27 @@ class UserData: ObservableObject {
             articleShoutoutsCounter = shoutoutsCounter
         }
     }
+    
+    func set(deviceToken: String) {
+        UserDefaults.standard.setValue(deviceToken, forKey: deviceTokenKey)
+    }
+    
+    func createUser() {
+        if let deviceToken = UserDefaults.standard.string(forKey: deviceTokenKey) {
+            print("Creating user with deviceToken \(deviceToken) and following \(followedPublicationIDs)...")
+            cancellables[.createUser] = Network.shared.publisher(for: CreateUserMutation(deviceToken: deviceToken, followedPublicationIDs: followedPublicationIDs))
+                .map { $0.user.uuid }
```

Comment:
```suggestion
                .map(\.user.uuid)
```

Reply:
This change produces error: `Contextual type for closure argument list expects 1 argument, which cannot be implicitly ignored`